app/src/main/java/ru/flicksbox/user/domain/UserRepository.kt

```diff
@@ -4,5 +4,7 @@ import kotlinx.coroutines.flow.Flow
 import ru.flicksbox.data.Data
 
 interface UserRepository {
-    fun getUser(cookie: String): Flow<Data<UserEntity>>
+    fun getUser(): Flow<Data<UserEntity>>
+    fun signup(username: String, email: String, password: String, repeated_password: String): Flow<Data<UserEntity>>
```

Comment:
`repeated_password` does not conform with code style, use camelCase: `repeatedPassword`